src/ext-test/java/org/opentripplanner/ext/siri/SiriAlertsUpdateHandlerTest.java

```diff
@@ -89,7 +95,7 @@ public void testSiriSxUpdateForStop() {
     ptSituation.setReportType(reportType);
 
     final SeverityEnumeration severity = SeverityEnumeration.SEVERE;
-    ptSituation.setSeverity(SeverityEnumeration.SEVERE);
+    ptSituation.setSeverity(severity);
```

Comment:
The variable `severity` should also be used on line 113 like this:

```
assertEquals(severity, transitAlert.severity);
```

Reply:
The other one is a Siri enum, while the other is an OTP enum, so they are not comparable.

Reply:
Then the variable should be inlined, it is used in only one place.